open-company/open-company-bot
# Move out Slack namespace oc.lib and some general cleanup

This PR is in support of: https://github.com/open-company/open-company-interaction/pull/3

This PR removes the Slack namespace that was moved to oc.lib to be shared with the Interaction service. It also cleans up the Bot system startup to be the same as the other services.

To test:

- Start the system with the normal, `lein start`. Does it start? Good. Kill it.
- Start the system instead with `lein repl` and `(go)`, does it start? Great. Keep it running.
- Ensure you are running with the latest pull of Storage service `phase3` branch and share an update to Slack
- Did the bot, run from the REPL, do its thing? Awesome.
- Stop the bot at the REPL with `(stop)`
- Share another update. Nothing happens? Bingo!
- Merge
- Deploy
- Celebrate the 35th anniversary of the liberation of the Falklands
